server/db/brave_hubs.csv.example

```diff
@@ -0,0 +1,3 @@
+system_id,system_name,heartbeat_alert_recipients

```

Comment:
It would be nice if this file name matched the script name. So be `add_hubs.csv.example`. That way it would be nice and easy to find because they will be beside each other in the directory structure. Might also be worth mentioning in the script's help text that this example file exists.

Reply:
i agree with this